microcosm_flask/routing.py

```diff
@@ -77,8 +77,10 @@ def decorator(func):
 
             if enable_metrics or ns.enable_metrics:
                 from microcosm_flask.metrics import StatusCodeClassifier
-                func = graph.metrics_counting(endpoint, classifier_cls=StatusCodeClassifier)(func)
-                func = graph.metrics_timing(endpoint)(func)
+                func = graph.metrics_counting("microcosm_flask",
```

Comment:
Feedback: is `microcosm_flask` a good name for the metric? Would we want a `microcosm_flask_route`? Something else?

Reply:
Summarizing slack discussion: `route` for name, `backend_type=microcosm_flask` tag.